frontend/src/components/Collection/components/CollectionDatasetsGrid/components/Row/DatsetRow/components/DeleteDataset/index.tsx

```diff
@@ -12,16 +11,12 @@ const AsyncAlert = loadable(
 );
 
 interface Props {
-  Button?: React.ElementType;
+  Button: ElementType;
```

Comment:
The `DeleteDataset` component is now only used by the dataset row "more" menu. The `Button` prop is now updated to a SDS component, and therefore the `Button` prop is changed from `optional` to `required`, thereby allowing us to remove the BlueprintJs `DefaultButton`.